vnext/ExperimentalFeatures.props

```diff
@@ -1,6 +1,6 @@
 <Project>
-  <PropertyGroup Condition="'$(SolutionName)'=='Microsoft.ReactNative.ProjectReunion'">
+  <PropertyGroup Condition="'$(SolutionName)'=='Microsoft.ReactNative.WindowsAppSDK'">
     <UseWinUI3>true</UseWinUI3>
-    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.22000.0</WindowsTargetPlatformVersion>
```

Comment:
Why this version?
If I recall correctly, the "default" current Win SDK version is `10.0.19041.0`.

Reply:
There's a bug (see: https://github.com/microsoft/microsoft-ui-xaml/issues/5828) in MIDL where it doesn't understand [experimental] correctly, and it was fixed in 22000 SDK. @asklar, now that we're on a stable release could we/would we want to use 19041?

Reply:
we can try :) If it works I see no problem with it

Reply:
did this work out @AgneLukoseviciute ?

Reply:
went down to 18362 if that's ok, saw this -> https://github.com/microsoft/react-native-windows/pull/8064/files#r656442369